cmd/queue/main.go

```diff
@@ -62,7 +62,7 @@ const (
 	badProbeTemplate   = "unexpected probe header value: %s"
 	failingHealthcheck = "failing healthcheck"
 
-	healthURLTemplate = "http://127.0.0.1:%d"
+	healthURLTemplate = "http://127.0.0.1:"
```

Comment:
I'm going to be a total pedant and point out that this isn't really a "template" any more :p. Maybe `healthURLPrefix`?

Reply:
sure why not :)

Reply:
though one can argue that prefix is a nil-adic template :)

Reply:
😂 😂